### ISSUE TITLE
Problem with Remove Adapter Message Binding

### ISSUE THREAD

**Issue by ligadata-yasser** [LigaData/Kamanja#1048]
I am using hash map as storage adapter
when listing the available adapters i can see this one among the list:
```
 {
  "APIResults" : {
    "Status Code" : 0,
    "Function Name" : "ListAllAdapterMessageBindings",
    "Result Data" : "testsmartinput_1,com.ligadata.kamanja.samples.messages.msg1,com.ligadata.kamanja.serializer.csvserdeser",
    "Result Description" : ""
  }
```

however when trying to remove using the command 

$KAMANJA_HOME/bin/kamanja $KAMANJA_HOME/config/MetadataAPIConfig.properties remove adaptermessagebinding KEY  'testsmartinput_1,com.ligadata.kamanja.samples.messages.msg1,com.ligadata.kamanja.serializer.csvserdeser'

i get:

```
Result: {
  "APIResults" : {
    "Status Code" : -1,
    "Function Name" : "StartMetadataAPI",
    "Result Data" : null,
    "Result Description" : "No value found for 'removeadaptermessagebindingtestsmartinput_1,com.ligadata.kamanja.samples.messages.msg1,com.ligadata.kamanja.serializer.csvserdeser'"
  }
}
```

**Comment by ligadata-rich** [comment]
This is a known issue, there is a fix coming for this.  During the flurry of development with a number of us changing the StartMetadataAPI (the front door to the kamanja command processing), some bad things happened to remove <just about anything>

The fix is in branch issue1037.   The following sorts of commands will work once this branch is merged with 1.4.0.Test:

$KAMANJA_HOME/bin/kamanja $apiConfigProperties remove container com.ligadata.kamanja.samples.containers.smokecodes.000000000001000000 TENANTID tenant1
$KAMANJA_HOME/bin/kamanja $apiConfigProperties remove message com.ligadata.kamanja.samples.messages.inpatientclaim.000000000001000000 TENANTID tenant1
$KAMANJA_HOME/bin/kamanja $apiConfigProperties remove model com.ligadata.kamanja.samples.models.copddataingest.000000000000000001 TENANTID tenant1
$KAMANJA_HOME/bin/kamanja $apiConfigProperties remove adaptermessagebinding key 'medicalinput,com.ligadata.kamanja.samples.messages.copdinputmessage,com.ligadata.kamanja.serializer.csvserdeser'

$KAMANJA_HOME/bin/kamanja $apiConfigProperties get container com.ligadata.kamanja.samples.containers.envcodes.000000000001000000 TENANTID tenant1
$KAMANJA_HOME/bin/kamanja $apiConfigProperties get message com.ligadata.kamanjabase.kamanjastatusevent.000000000000000001 TENANTID tenant1
$KAMANJA_HOME/bin/kamanja $apiConfigProperties get model com.ligadata.kamanja.samples.models.copdriskassessment.000000000000000001 TENANTID tenant1
$KAMANJA_HOME/bin/kamanja $apiConfigProperties get type system.float.1000000
$KAMANJA_HOME/bin/kamanja $apiConfigProperties get function pmml.AnyBetween.1

In addition to remove, there were problems with the "get" commands as well.

Pokuri is reviewing the change and once vetted will merge it to 1.4.0.Test

**Comment by ligadata-yasser** [comment]
verified
